src/analysis/retail/evoker/preservation/normalizers/CastLinkNormalizer.ts

```diff
@@ -84,6 +193,52 @@ const EVENT_LINKS: EventLink[] = [
       );
     },
   },
+  /* SEEDLING LINKING */
+  // link seedling heal to Emerald Blossom echo heal
+  {
+    linkRelation: FLUTTERING_SEEDLINGS_ECHO,
+    reverseLinkRelation: FLUTTERING_SEEDLINGS_ECHO,
+    linkingEventId: SPELLS.EMERALD_BLOSSOM_ECHO.id,
+    linkingEventType: EventType.Heal,
+    referencedEventId: SPELLS.FLUTTERING_SEEDLINGS_HEAL.id,
+    referencedEventType: EventType.Heal,
+    anyTarget: true,
+    forwardBufferMs: EB_BUFFER_MS,
+    maximumLinks(c) {
+      return c.getTalentRank(TALENTS_EVOKER.FLUTTERING_SEEDLINGS_TALENT);
+    },
+    isActive(c) {
+      return c.hasTalent(TALENTS_EVOKER.FLUTTERING_SEEDLINGS_TALENT);
+    },
+    additionalCondition(linkingEvent, referencedEvent) {
+      return (
+        !HasRelatedEvent(referencedEvent, FLUTTERING_SEEDLINGS_ECHO) &&
```

Comment:
Should this be `FLUTTERING_SEEDLINGS_HARDCAST` ?

Reply:
Ya